lib/bigquery/job.js

```diff
@@ -156,35 +188,9 @@ nodeutil.inherits(Job, ServiceObject);
  * @example
  * job.cancel(function(err, apiResponse) {
  *   // Check to see if the job completes successfully.
- *   onJobComplete(function(err) {
- *     if (!err) {
- *       // Job cancelled successfully.
- *     }
- *   });
+ *   job.on('error', function(err) {});
+ *   job.on('complete', function(metadata) {});
```

Comment:
Pretty sure I've asked this question in regards to operations before, but any reason we don't just add the event listeners internally so the usage is

```js
job.cancel(function(err, metadata) {});
```